src/status_im/common/home/actions/view.cljs

```diff
@@ -277,23 +277,29 @@
           :sub-label           nil
           :chevron?            false}))
 
-;; TODO(OmarBasem): Requires design input.
 (defn show-qr-entry
-  []
+  [public-key]
   (entry {:icon                :i/qr-code
           :label               (i18n/label :t/show-qr)
-          :on-press            #(js/alert "TODO: to be implemented, requires design input")
+          :on-press            (fn []
+                                 (rf/dispatch [:universal-links/generate-profile-url
+                                               {:public-key public-key
+                                                :on-success #(rf/dispatch [:open-modal
+                                                                           :share-contact])}]))
           :danger?             false
           :accessibility-label :show-qr-code
           :sub-label           nil
           :chevron?            false}))
 
-;; TODO(OmarBasem): to be implemented.
 (defn share-profile-entry
-  []
+  [public-key]
   (entry {:icon                :i/share
           :label               (i18n/label :t/share-profile)
-          :on-press            #(js/alert "TODO: to be implemented")
+          :on-press            (fn []
+                                 (rf/dispatch [:universal-links/generate-profile-url
+                                               {:public-key public-key
+                                                :on-success #(rf/dispatch [:open-share
+                                                                           {:options {:message %}}])}]))
```

Comment:
Shouldn't share-profile share the account public key key using `react-native/share `?